packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -2360,6 +2383,25 @@ export class Grapher
         else if (renderWidth >= 1080) this.baseFontSize = 18
     }
 
+    @computed get isNarrow(): boolean {
+        if (this.isExportingtoSvgOrPng) return false
```

Comment:
Note for the reviewers: For now, exported charts use ideal bounds. That's why they can never be narrow, for example. But when @marcelgerber presented our new cloudflare workers, it seemed like we wanted static charts that could potentially have any size?

Reply:
okay, thank you, both!